script.cu.lrclyrics/changelog.txt

```diff
@@ -1,3 +1,14 @@
+v6.3.1
```

Comment:
From what I recall, we've been asking folks not to include a changelog in final submissions because they aren't used anymore (at least for Matrix and, I think, Leia).

Reply:
as i've said before, imo that was a bad advice. 
it's recommended to include a changelog with any piece of software.